APSetup/Files/etc/hostapd/hostapd.conf

```diff
@@ -16,7 +16,7 @@ rsn_pairwise=CCMP
 # Configuration for an AP on wlan1 (5G)
 interface=wlan1
 driver=nl80211
-ssid=CSNA
+ssid=Phoebus
```

Comment:
This is extremely minor but should Pheobus be caps?

Reply:
I dont think it matters. Its just based on what you want the SSID to look like when its broadcasted

Reply:
Gotcha, this'll be changeable by the user anyways